app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.app.administratorcontrols
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+const val ADMINISTRATOR_CONTROLS_ACTIVITY_PROFILE_ID_ARGUMENT_KEY = "AdministratorControlsActivity.profile_id"
+
+/** Activity for Administrator Controls. */
```

Comment:
Similar to this add class kDoc to other files to like presenter and fragments.

Reply:
Done